### PR TITLE
Add PQ Braking and ACC led control CAN messages

### PR DESCRIPTION
**Pull request by CineXWorm**
mMotor_Bremse used to control braking on cars that don't support ACC.
mAWV used to control ACC led in dashboard.

### CODE REVIEW

**Comment by grekiki** [comment]
Would help to get this merged into openpilot to be able to tune pq long with plotjuggler, seems like it doesn't take local DBCs

**Comment by grekiki** [comment]
Would be really nice to have this merged for using Plotjuggler for tuning long control on PQ. Seems like plotjuggler doesn't use the local DBC(or I am just doing something wrong) @adeebshihadeh

**Comment by adeebshihadeh** [comment]
LGTM. any comments @jyoung8607?

@grekiki yeah, it should use the local DBC

File: vw_golf_mk4.dbc [hunk 1]
```diff
@@ -1138,6 +1138,39 @@ BO_ 210 HCA_1: 5 XXX
  SG_ LM_OffSign : 31|1@1+ (1,0) [0|1] "" XXX
  SG_ Vib_Freq : 32|4@1+ (4,0) [0|60] "Hz" XXX
  SG_ Vib_Amp : 36|4@1+ (0.5,0) [0|7.5] "Nm" XXX
+ 
+BO_ 644 mMotor_Bremse: 6 Motor_Diesel
```
Comment (jyoung8607) [inline vw_golf_mk4.dbc:1142]:
```suggestion
BO_ 644 Motor_Bremse: 6 XXX
```

The prefixing `m` on message names is unique to the source of this data, we have been removing them.

File: vw_golf_mk4.dbc [hunk 2]
```diff
@@ -1138,6 +1138,39 @@ BO_ 210 HCA_1: 5 XXX
...
+ SG_ MOB_Freigabe : 14|1@1+ (1,0) [0|1] "" BCM_Gateway,Bremse_MK25AESP,Gateway_separat
+ SG_ MOB_Anhaltewunsch : 13|1@1+ (1,0) [0|1] "" Bremse_MK25AESP,Getriebe_DQ
+ SG_ MOB_CHECKSUM : 0|8@1+ (1,0) [0|255] "" Bremse_MK25AESP
+ SG_ MOB_COUNTER : 8|4@1+ (1,0) [0|15] "" Bremse_MK25AESP
+ SG_ TSK_v_Begrenzung_aktiv : 15|1@0+ (1,0) [0|1] "" XXX
+ SG_ TSK_ax_Getriebe_01 : 40|8@1+ (0.048,0) [0|255] "m/s2" XXX
+ SG_ MOB_Bremsstgr : 16|11@1+ (0.76,0) [0|100] "Unit_PerCent" Vector__XXX
+ SG_ MOB_Bremsmom : 27|13@1+ (1,0) [0|8190] "Unit_NewtoMeter" Bremse_MK25AESP
+
+BO_ 870 mAWV: 5 ACC
```
Comment (jyoung8607) [inline vw_golf_mk4.dbc:1153]:
```suggestion
BO_ 870 AWV: 5 XXX
```

Same as above

File: vw_golf_mk4.dbc [hunk 3]
```diff
@@ -1138,6 +1138,39 @@ BO_ 210 HCA_1: 5 XXX
  SG_ LM_OffSign : 31|1@1+ (1,0) [0|1] "" XXX
  SG_ Vib_Freq : 32|4@1+ (4,0) [0|60] "Hz" XXX
  SG_ Vib_Amp : 36|4@1+ (0.5,0) [0|7.5] "Nm" XXX
+ 
+BO_ 644 mMotor_Bremse: 6 Motor_Diesel
+ SG_ MOB_Standby : 12|1@1+ (1,0) [0|1] "" Bremse_MK25AESP
```
Comment (jyoung8607) [inline vw_golf_mk4.dbc:1143]: [ example only ]

```suggestion
 SG_ MOB_Standby : 12|1@1+ (1,0) [0|1] ""  XXX
```

Let's lose the network-node source/consumer labels unless and until we're willing to spend the time overhauling the file to make good use of them. No real sense in adding a few PQ25-specific dangling loose references without the other DBC infrastructure for DBC editors to even make use of them. I admit to having done this inconsistently in the past but we'll clean it all up eventually.

**Review by jyoung8607 — CHANGES_REQUESTED** [review]

**Pull request merged**